### DISCUSSION TITLE
Badaso v3.x (Buayo)

### DISCUSSION THREAD

**Discussion by rizkiheryandi** [uasoft-indonesia/badaso#394] [Show and tell]
Hello guys,

We will start to develop badaso v3.x called Harimau

Why we start it ?

- Badaso is a new comers tools, so it's need newest technology too

What changes will be implement on badaso v3.x ?

- Vue 3 support
- Use typescript
- Replace vue-router with [inertia.js](https://github.com/inertiajs/inertia)
- Use [vuetail](https://github.com/uasoft-indonesia/vuetail)
- Laravel octane ready 
- You can check the full roadmap [here](https://github.com/uasoft-indonesia/badaso/projects/4) 

If you have any question or idea, please comment

Thank you in advance

**Comment by Dri372** [comment]
Hi @rizkiheryandi 

Thanks for your job.

You can implement this feature https://github.com/uasoft-indonesia/badaso/issues/620 (if sb wants it in Badaso he can easily patch the package)

which can be an option per Crud.
